serving/docker/partition/properties_manager.py

```diff
@@ -152,6 +158,14 @@ def set_and_validate_entry_point(self):
                     engine = self.properties.get('engine')
                     if engine is None:
                         raise ValueError("Please specify engine")
+                    elif quantize:
+                        if engine.lower() == "mpi":
+                            entry_point = "djl_python.huggingface"
```

Comment:
We could do assign entrypoint to huggingface, only when vllm or lmi-dist SERVING_FEATURES env variable.